content/docs/operating/integrations.md

```diff
@@ -69,6 +69,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [AWS SNS](https://github.com/DataReply/alertmanager-sns-forwarder)
   * [DingTalk](https://github.com/timonwong/prometheus-webhook-dingtalk)
   * [GELF](https://github.com/b-com-software-basis/alertmanager2gelf)
+  * [Icinga2](https://github.com/vshn/signalilo): creates services in Icinga2 based on received notifications
```

Comment:
```suggestion
  * [Icinga2](https://github.com/vshn/signalilo)
```

Reply:
Oh ok, I've adjusted that in the latest version of the commit.